test/functional/mempool_unbroadcast.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that the mempool ensures transaction delivery by periodically sending
+to peers until a GETDATA is received."""
+
+from io import BytesIO
+import time
+
+from test_framework.blocktools import create_block, create_coinbase
+from test_framework.messages import ToHex, CTransaction
+from test_framework.mininode import P2PTxInvStore
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+        assert_equal,
+        disconnect_nodes,
+        connect_nodes,
+        create_confirmed_utxos,
+        wait_until,
+        hex_str_to_bytes
```

Comment:
in commit f40f6875f5570cb25882d8a04a6cbeb61ee7366d:

Style nit. Could add trailing comma for symmetry, git blame, and code formatters?

```suggestion
        hex_str_to_bytes,
```

Reply:
@amitiuttarwar Since this is a new file, you might want to run [`black`](https://black.readthedocs.io/en/stable/) on the entire file to get proper formatting which addresses MarcoFalke's nit and all other formatting nits :)

Reply:
@practicalswift 
Style is subjective and we allow the pull request author to pick the style, as long as it is reasonable. We don't require a specific python formatter. While black or yapf (https://github.com/bitcoin/bitcoin/blob/master/.style.yapf) or the built-in formatter of your editor can be used, they are not mandatory.

Reply:
@MarcoFalke Have I claimed anything else? :)  This is _your_ style nit - I don't care :)

Reply:
fixed! thanks for the style tips